frontend/src/metabase/admin/settings/components/SettingsPages/GeneralSettingsPage.tsx

```diff
@@ -0,0 +1,75 @@
+import { jt, t } from "ttag";
+
+import { useDocsUrl } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Stack } from "metabase/ui";
+
+import { AdminSettingInput } from "../widgets/AdminSettingInput";
+import { AnonymousTrackingInput } from "../widgets/AnonymousTrackingInput";
+import { CustomHomepageDashboardSetting } from "../widgets/CustomHomepageDashboardSetting";
+import { HttpsOnlyWidget } from "../widgets/HttpsOnlyWidget";
+import { SiteUrlWidget } from "../widgets/SiteUrlWidget";
+
+export function GeneralSettingsPage() {
+  const { url: iframeDocsUrl } = useDocsUrl("configuring-metabase/settings", {
+    anchor: "allowed-domains-for-iframes-in-dashboards",
+  });
+
+  return (
+    <Stack gap="xl" maw="42rem" px="lg" py="sm">
```

Comment:
I'm a little conflicted about whether I should extract this as an `AdminSettingsWrapper` component just for the sake of keeping it consistent across pages

Reply:
If it's going to be the same for all these pages I think we should, otherwise we'll just have `<Stack gap="xl" maw="42rem" px="lg" py="sm">` 13 times.

Is this specific to the `Settings` or is it for any Admin tab like `Performance` and `Troubleshooting` as well?

Reply:
Well, as I've started another page, I've already found the need to tweak this (max width) a bit, so let's leave it for now, we can revisit later after we've converted most/all the pages